.gitignore

```
@@ -46,3 +46,5 @@ node_modules/
 coverage/
 .nyc_output/
 package-lock.json
+# Eclipse Buildship files
+**/.project
```

This will match the same files:
```suggestion
.project
```
From https://git-scm.com/docs/gitignore#_pattern_format:
> If there is a [directory] separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular `.gitignore` file itself. Otherwise the pattern may also match at any level below the `.gitignore` level.

---

I'm with Raphael, to avoid accidentally merging, I'm pressing the "request changes" button.